src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -118,7 +118,7 @@ def repo_status_icon(status, details = nil)
 
     description = REPO_STATUS_DESCRIPTIONS[status] || 'Unknown state of repository'
     description = 'State needs recalculations, former state was: ' + description if outdated
-    description << " (" + details + ")" if details
+    description += " (" + details + ")" if details
```

Comment:
how can that make any difference? They are exactly the same the same thing.

``` Ruby
irb(main):013:0> a = "hola "
=> "hola "
irb(main):014:0> a += "mundo"
=> "hola mundo"
irb(main):015:0> a
=> "hola mundo"
irb(main):016:0> b = "hola "
=> "hola "
irb(main):017:0> b << "mundo"
=> "hola mundo"
irb(main):018:0> b
=> "hola mundo"
```
https://ruby-doc.org/core-2.2.0/String.html#method-i-3C-3C